### ISSUE TITLE
Some questions regarding your work, thanks.

### ISSUE THREAD

**Issue by huaiyanggongzi** [Time-MoE/Time-MoE#83]
Hi Dr.Shi, 

Thank you very much for sharing your wonderful book. I have several questions regarding training your model using my own data.

1) what are the relationship between global batch size and micro batch size. Is global_batch_size = micro_batch_size * number of GPU?  Are there any recommendations for setting up the size of micro_batch_size?

2) In the training parameter, there is one term as "max_length", is that the same as "context_length", or "context_length" +"prediction_length"

3) In terms of evaluation, I noticed that you applied "mean, std" for normalizing the context sequence? Before preparing sequences for training, do we need to apply this kind of normalization techniques?

4) I have many sequences of different lengths, from 500 to 3000? Should I just keep longer ones? If I want to use all of them, should I control the max_length parameter to be smaller enough? Is concatenating all these small sequences into longer one a good strategy? 

Thank you very much for your help.

**Comment by Maple728** [comment]
Thank you for your interest in our work!

1. You are correct! The `global_batch_size` is equal to `micro_batch_size` multiplied by the number of GPUs. We recommend setting `micro_batch_size` to the largest possible value until reaching the GPU memory limit.  
2. The `max_length` is the sum of `context_length` and `prediction_length`.  
3. To apply zero-normalization, simply pass `--normalization zero` to `main.py`, and the process will be handled automatically.  
4. You can keep all sequences as they are and set `max_length` to your desired value. In fine-tuning mode, Time-MoE will split sequences longer than `max_length` into multiple sub-sequences and pad shorter sequences at the tail. Concatenating all sequences into longer ones improves computational efficiency, which is why we use this approach during Time-MoE’s pretraining phase.  

Please feel free to contact us if you have any questions.
